### ISSUE TITLE
Crash report - Index 1 is out of bound

### ISSUE THREAD

**Issue by danielhopkins** [github/hub#839]
Error (*errors.errorString): `Index 1 is out of bound`

Stack:

```
goroutine 1 [running]:
github.com/github/hub/commands.(*Args).InsertParam(0xc2080112d0, 0x1, 0xc208081d38, 0x4, 0x4)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/args.go:95 +0x170
github.com/github/hub/commands.transformCheckoutArgs(0xc2080112d0, 0x0, 0x0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/checkout.go:102 +0xd5b
github.com/github/hub/commands.checkout(0x6826a0, 0xc2080112d0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/checkout.go:36 +0x40
github.com/github/hub/commands.(*Command).Call(0x6826a0, 0xc2080112d0, 0x0, 0x0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/commands.go:46 +0x189
github.com/github/hub/commands.(*Runner).Call(0xc20802b120, 0x6826a0, 0xc2080112d0, 0x0, 0x0, 0x0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/runner.go:88 +0x54
github.com/github/hub/commands.(*Runner).Execute(0xc20802b120, 0x0, 0x0, 0x0)
	/var/folders/r7/cp0y2kgs15l6yc67v8_jy1j80000gn/T/go/src/github.com/github/hub/commands/runner.go:80 +0x1e4
main.main()
	/private/tmp/hub-lKNcwz/main.go:12 +0x40

```

Runtime:

```
GOOS: darwin
GOARCH: amd64
```

**Comment by danielhopkins** [comment]
Done while doing a git checkout -t from a private repo

**Comment by jingweno** [comment]
This bug is fixed in https://github.com/github/hub/commit/2884da15b2a73097d5b51dc40aa047e1ef2ec56b. We'll make a new release soon.

**Comment by jingweno** [comment]
This bug will be fixed in v2.2.1. Closing.
